### ISSUE TITLE
CTkProgressBar.configure(corner_radius) triggers exception 

### ISSUE THREAD

**Issue by avalon60** [TomSchimansky/CustomTkinter#584]
CustomTkinter 4.6.3

Adding this third line, to simple_example.py, causes the exception:

```
progressbar_1 = customtkinter.CTkProgressBar(master=frame_1)
progressbar_1.pack(pady=12, padx=10)
progressbar_1.configure(corner_radius=25)
```

Here is the stack:

```
Traceback (most recent call last):
  File "/home/clive/PycharmProjects/customtk/simple_example.py", line 39, in <module>
    progressbar_1.configure(corner_radius=25)
  File "/home/clive/PycharmProjects/customtk/venv/lib/python3.8/site-packages/customtkinter/widgets/ctk_progressbar.py", line 166, in configure
    super().configure(require_redraw=require_redraw, **kwargs)
  File "/home/clive/PycharmProjects/customtk/venv/lib/python3.8/site-packages/customtkinter/widgets/widget_base_class.py", line 131, in configure
    super().configure(**kwargs)
  File "/usr/lib/python3.8/tkinter/__init__.py", line 1646, in configure
    return self._configure('configure', cnf, kw)
  File "/usr/lib/python3.8/tkinter/__init__.py", line 1636, in _configure
    self.tk.call(_flatten((self._w, cmd)) + self._options(cnf))
_tkinter.TclError: unknown option "-corner_radius"
```

**Comment by Akascape** [comment]
Duplicate of https://github.com/TomSchimansky/CustomTkinter/issues/579

@avalon60 Corner_radius is not added in the configure settings:

For these we have to use `progressbar_1.corner_radius=0` in order to change it.

**Comment by TomSchimansky** [comment]
Fixed with version 5.0.0.
